repo_id,repo_full_name,file,file_sha1,match
369865456,7h3f0x/exploits,CVE-2021-26567/POC.py,433dcb3da10fc90a7407b7db846784e19bee23f9,CVE-2021-26567
369865456,7h3f0x/exploits,CVE-2020-0609/exploit.py,88cd36b0194adffc21f011c513228c3a787a01c6,CVE-2020-0609
